### DISCUSSION TITLE
MoltenVK update for UE4 games

### DISCUSSION THREAD

**Discussion by demetrius1313** [Whisky-App/Whisky#358] [Q&A]
Hi! Is there anyway to update moltenvk version for playing UE4 games? Because current version showing black screen after launch

**Comment by rockyslaymaker** [comment]
replace /Users/user/Library/Application Support/com.isaacmarovitz.Whisky/Libraries/Wine/lib/libMoltenVK.dylib with the version you want to use. You might have to replace it every time whisky updates.
